packages/cosmic-swingset/src/extract-proposal.js

```diff
@@ -58,20 +58,18 @@ export const extractCoreProposalBundles = async (
     .stat(dirname)
     .then(stbuf => (stbuf.isDirectory() ? dirname : path.dirname(dirname)));
 
-  const bundleToSource = new Map();
-  const bundleHandles = new Set();
-  const makeCPArgs = await Promise.all(
+  const extracted = await Promise.all(
     coreProposals.map(async (initCore, i) => {
+      const bundleToSource = new Map();
+      const bundleHandles = new Set();
+
```

Comment:
Is it possible that different proposals may share a bundle?
From what I gather, before `bundleToSource` was shared, so if the same absolute bundle path came up in multiple proposals, the source had to match.

Since `bundleToSource` doesn't seem to impact output and is just there to ensure there are no collisions, is there a reason to not keep it shared across proposals?